client/src/components/productOverview/ProductInfo.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types */
+/* eslint class-methods-use-this: ["error", { "exceptMethods": ["render"] }] */
+import React from 'react';
+import { connect } from 'react-redux';
+
+import StarRating from '../common/starRating.jsx';
+import ProductTitle from './ProductTitle.jsx';
+import ProductStyle from './ProductStyle.jsx';
+import SelectSize from './SelectSize.jsx';
+import SelectQty from './SelectQty.jsx';
+
+class ProductInfo extends React.Component {
+  render() {
+    return (
+      (this.props.styles.length === 0)
+        ? <div>loading...</div>
```

Comment:
Love this!
I was just thinking about adding a loading component to visually inform the user that sth is in progress. Thanks for adding it in!